lib/features/rules_filter_creator/presentation/model/rule_filter_action_arguments.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:equatable/equatable.dart';
+import 'package:tmail_ui_user/features/rules_filter_creator/presentation/model/email_rule_filter_action.dart';
+
+class RuleFilterActionArguments with EquatableMixin {
+  final EmailRuleFilterAction? action;
+  final dynamic detailed;
```

Comment:
Use of `dynamic` is not recommended. You should specify a specific style for it. Furthermore, the variable name `detailed` is unclear.